src/components/Exercise/ExerciseAdd.js

```diff
@@ -0,0 +1,108 @@
+import React, { useRef } from 'react'
+import styled from 'styled-components/macro'
+import pageBackIcon from '../icons/back.png'
+import { Link } from 'react-router-dom'
+import Button from '../utils/Button'
+import useForm from '../utils/hooks/useForm'
+
+export default function ExerciseAdd({ handleExerciseAdd }) {
+  const inputEl = useRef(null)
+
+  const handleAdd = () => {
+    handleExerciseAdd(inputs.name, inputs.type, inputs.instructions)
+  }
+  const { inputs, handleInputChange, handleSubmit } = useForm(handleAdd)
+
+  return (
+    <ExerciseGrid onSubmit={handleSubmit}>
+      <ButtonWrapper>
```

Comment:
Würde es Button in Mehrzahl schreiben -> Buttons, da du ja den Wrapper für mehrere Buttons nutzt.